examples/hosts/app-integration/external-data/tests/utilities.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Server } from "http";
+
+/**
+ * "Promisifies" `Server.close`.
+ */
+export async function closeServer(server: Server): Promise<void> {
+    await new Promise<void>((resolve) => {
+        server.close(() => {
+            resolve();
+        });
+    });
```

Comment:
What happens if `server.close()` throws? I don't recall if that will automatically be "wrapped" in a rejection of the Promise. Recently ran into issues with Promises that might resolve but never reject if an error occurs :).

Reply:
The callback function receives an error in that case. I had opted to ignore it while I was developing, but it would be good for our tests to capture if that happens. Will update.